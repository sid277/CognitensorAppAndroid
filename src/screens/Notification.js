import React from 'react';
import { Text, StyleSheet } from 'react-native';
import DefaultScrollView from '../components/default/DefaultScrollView';

const Notification = () => {
  return (
    <DefaultScrollView
      styleView={styles.container}
      styleScroll={styles.scrollContainer}>
      <Text>No Notifications</Text>
    </DefaultScrollView>
  );
};

const styles = StyleSheet.create({
  container: {
    backgroundColor: '#FFFFFF',
  },
  scrollContainer: {
    alignItems: 'center',
    justifyContent: 'center',
  },
});

export default Notification;
