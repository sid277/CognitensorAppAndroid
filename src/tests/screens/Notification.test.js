import React from 'react';
import { shallow } from 'enzyme';
import Notification from '../../screens/Notification';
import renderer from 'react-test-renderer';


test('should test Notification component', () => {
const wrapper = shallow(<Notification />);
expect(wrapper).toMatchSnapshot();
});