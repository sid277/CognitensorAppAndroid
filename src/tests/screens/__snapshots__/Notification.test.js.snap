// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test Notification component 1`] = `ShallowWrapper {}`;
